### DISCUSSION TITLE
Prisma data proxy and regular connection

### DISCUSSION THREAD

**Discussion by dikkeduif** [prisma/prisma#15919] [Q&A] [answered]
Hey,

We've started using the prisma data proxy to connect to our planetscale database.

One thing we noticed is that depending on the amount of data that is being returned, data proxy fails to fetch this data. For example on specific query returns +4 MB of data back to the client. When going through the data proxy this fails, when not using the data proxy the query runs fine and returns the data normally.

Obviously we want to optimize this query first of all, but more importantly, the work around would be for me to skip the data proxy whenever we need to run these kinds of queries.

How can we achieve this?

I was hoping I could connect to the same database within prisma, once via the data proxy way, and once without, but I don't think it's possible to set this kind of connection up in the prisma schema.

Is there something I'm missing, is the data proxy connection really not able to run these kinds of queries?

Thanks

D

**Comment by vladi-stevanovic** [comment, marked as answer]
👋 Hi @dikkeduif thank you for raising this question! 
Could you please open a support ticket on the Prisma Data Platform with this question? See here the steps on how to [Contact Support](https://www.prisma.io/docs/data-platform/contact-support).

Our Community channels such as GitHub Discussions, Slack, and Stack Overflow are dedicated to answering questions about our OSS ORM (Prisma Client, Migrate, Studio).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
